README.md

```diff
@@ -0,0 +1,69 @@
+<a href="https://flotiq.com/">
+    <img src="https://editor.flotiq.com/fonts/fq-logo.svg" alt="Flotiq logo" title="Flotiq" align="right" height="60" />
+</a>
+
+Flotiq Mobile Expo
+========================
+
+This is a mobile demo project for browsing Flotiq entries. It is configured to pull all data from your [Flotiq](https://flotiq.com) account.
+
+You can use this React Native project to kickstart your next mobile app development.
+
+Install the app from Apple App Store or Google Play, use the QR code in your API keys to connect the app to Flotiq and browse through your content.
+
+[![Flotiq Mobile Expo on Apple App Store](https://user-images.githubusercontent.com/551004/29770691-a2082ff4-8bc6-11e7-89a6-964cd405ea8e.png)](https://apps.apple.com/app/flotiq-mobile-expo/id1505331246) [![Flotiq Mobile Expo on Google Play](https://user-images.githubusercontent.com/551004/29770692-a20975c6-8bc6-11e7-8ab0-1cde275496e0.png)](https://play.google.com/store/apps/details?id=com.flotiqmobiledemo)
+
+This project was generated using [Expo](https://expo.io) version 4.5.1 with [React Native](https://reactnative.dev/) SDK version 41.0.0 and [React](https://reactjs.org/) version 16.13.1 and was tested for Android 10 and 11.
+
```

Comment:
I think we should add more detailed info about the application, for example:

Key features:
- Login via API Key
- API Key QR code scanner
- Listing Content Type Definitions
- Listing Content Objects
- Single Content Object preview 
- Content Object management (create, update, delete)
- Content Objects search
- Adding media using the file browser or camera

Reply:
added